metis-indexing/src/main/java/eu/europeana/indexing/tiers/media/AbstractMediaClassifier.java

```diff
@@ -62,9 +62,11 @@ public final TierClassification<MediaTier, ContentTierBreakdown> classify(RdfWra
                                                           .orElse(MediaTier.T0);
       mediaResourceTechnicalMetadataList = descendingMediaResourceTechnicalMetadata;
     }
+    MediaType mediaTypeResult = getMediaType();
+    boolean hasMediaResource3DAvailable = mediaTypeResult == MediaType.THREE_D && (mediaTier != MediaTier.T0 && mediaTier != MediaTier.T1);
 
-    final ContentTierBreakdown contentTierBreakdown = new ContentTierBreakdown(getMediaType(), entityLicenseType, hasThumbnails,
-        hasLandingPage, hasEmbeddableMedia, mediaResourceTechnicalMetadataList);
+    final ContentTierBreakdown contentTierBreakdown = new ContentTierBreakdown(mediaTypeResult, entityLicenseType, hasThumbnails,
+        hasLandingPage, hasMediaResource3DAvailable,hasEmbeddableMedia, mediaResourceTechnicalMetadataList);
```

Comment:
We need to refactor this class ContentTierBreakdown the constructor is polluted with many boolean parameters. I think is not the scope of this story. Can you create a ticket to refactor this?

Reply:
I decided to handle this anyway and created a Builder.
This pollution should be gone now

Reply:
:+1: